### DISCUSSION TITLE
How to use Firebase emulator with Authentication ?

### DISCUSSION THREAD

**Discussion by gjpsiqueira** [invertase/react-native-firebase#4544] [Q&A] [answered]
I'm using the Firebase Emulator to testing my app React Native. So, I installed the Firebase Tools and started the emulator authentication, however, when I try set up on my app, i can't found the method: `firebase.auth().useEmulator('http://localhost:9099/');` , there's another way to that ?

**Comment by milesibastos** [comment, marked as answer]
https://github.com/invertase/react-native-firebase/discussions/4461

**Reply by mikehardy** [reply]
Indeed, "coming soon", although now that the big breaking change to port react-native-firebase to required versions is done, it is open for anyone to contribute if they want to implement it - I will happily help shepherd it to merge+release
